Changelog (#2663)
seqan/seqan3: CHANGELOG.md

# 3.0.3

Note that 3.1.0 will be the first API stable release and interfaces in this release might still change.

> 
> * Check out our new [SeqAn3 Cookbook](https://docs.seqan.de/seqan/3.0.3/cookbook.html). It contains a listing of code
>   examples on how to perform particular tasks using the library.

Reviewer: ```suggestion
```

It was in the 3.0.2 changelog because it was all new.
We can either remove it or rewrite it and advertise that it now has many more entries or something.

> * SeqAn 3.0.2 is known to compile with GCC 7.5, 8.4, 9.3, 10.3, and 11.1. Future versions (e.g. GCC 11.2 and 12) might

Reviewer: ```suggestion
* SeqAn 3.0.3 is known to compile with GCC 7.5, 8.4, 9.3, 10.3, and 11.1. Future versions (e.g. GCC 11.2 and 12) might
```

Author: good catch

>   work, but weren’t yet available at the time of this release.

Reviewer: ```suggestion
  work, but were not yet available at the time of this release.
```

I guess this was also "incorrect" in the other releases

Author: Should I fix those, too?

Reviewer: If you want to, but not that important

Author: done

#### Search

> * The `seqan3::fm_index_cursor` and `seqan3::bi_fm_index_cursor` can be serialised
> ([\#2048](https://github.com/seqan/seqan3/pull/2048)).

Reviewer 2: ```suggestion
* The `seqan3::fm_index_cursor` and `seqan3::bi_fm_index_cursor` can be serialised
  ([\#2048](https://github.com/seqan/seqan3/pull/2048)).
```
